test/widgets/autocomplete_test.dart

```diff
@@ -352,24 +352,24 @@ void main() {
       await tester.pumpAndSettle();
 
       // "topic three" and "topic two" appear, but not "topic one"
-      checkTopicShown(topic1, store, expected: false);
-      checkTopicShown(topic2, store, expected: true);
-      checkTopicShown(topic3, store, expected: true);
+      checkTopicShown('Topic one',   store, expected: false);
+      checkTopicShown('Topic two',   store, expected: true);
+      checkTopicShown('Topic three', store, expected: true);
```

Comment:
autocomplete test [nfc]: Use plain strings for finding topic
    
    instead of inferring it from a TopicName
    
    Signed-off-by: Zixuan James Li <zixuan@zulip.com>

Commit-message nit: I think "inferring" isn't quite the right word; if you have a `TopicName` and you want its display name, you just look at its `displayName` field; no inference is needed